src/wallet/rpcwallet.cpp

```diff
@@ -3161,6 +3161,39 @@ UniValue createwallet(const JSONRPCRequest& request)
     return obj;
 }
 
+static UniValue unloadwallet(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() > 1) {
+        throw std::runtime_error(
+            "unloadwallet ( \"wallet_name\" )\n"
+            "Unloads the wallet specified in the argument otherwise unloads the wallet referenced by the request endpoint.\n"
+            "\nArguments:\n"
+            "1. \"wallet_name\"    (string, optional) The name of the wallet to unload.\n"
+            "\nExamples:\n"
+            + HelpExampleCli("unloadwallet", "wallet_name")
+            + HelpExampleRpc("unloadwallet", "wallet_name")
+        );
+    }
+
+    std::shared_ptr<CWallet> wallet;
+
+    if (request.params[0].isNull()) {
+        wallet = GetWalletForJSONRPCRequest(request);
+    } else {
+        wallet = GetWallet(request.params[0].get_str());
+    }
+
+    if (!wallet) throw JSONRPCError(RPC_WALLET_NOT_FOUND, "Requested wallet does not exist or is not loaded");
+
+    RemoveWallet(wallet);
+    UnregisterValidationInterface(wallet.get());
+    wallet->BlockUntilSyncedToCurrentChain();
+    wallet->Flush(true);
+    wallet->Unload();
```

Comment:
In commit *rpc: Add unloadwallet RPC*, this doesn't build (`CWallet.Unload()` is only added in *ui: Support wallets unloaded dynamically*)

Reply:
Fixed.